### DISCUSSION TITLE
Use GitHub code search in VSCode

### DISCUSSION THREAD

**Discussion by Milo123459** [community/community#14807] [Other Feature Feedback, Questions, & Ideas] [answered]
A VSCode extension to use GitHub code search could be quite interesting.

**Comment by look** [comment, marked as answer]
This is on our radar. Previous feedback for a VS Code extension:

* [Integrate Code Search with VS Code](https://github.com/github/feedback/discussions/8798)
* [VS Code Extension plz?](https://github.com/github/feedback/discussions/9994)

**Reply by Anselmoo** [reply]
I really like the [Sourcegraph VSCode-Integration](https://marketplace.visualstudio.com/items?itemName=sourcegraph.sourcegraph) because it allows opening the searched code straight in the IDE.

**Comment by orrin-naylor-instacart** [comment]
Any updates on this. For larger repos, like the one at instacart, the native search in VSCode is completely useless.
